canonical/dqlite.io
# Update nav with new branding

## Done

- Update nav with new branding

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8037
- Check that the global nav works as expected


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5635
